packages/electrode-archetype-react-app-dev/config/archetype.js

```diff
@@ -74,6 +74,7 @@ const karmaConfigSpec = {
 
 const babelConfigSpec = {
   enableTypeScript: { env: "ENABLE_BABEL_TYPESCRIPT", default: false },
+  enableDynamicImport: { env: "ENABLE_DYNAMIC_IMPORT", default: true },
```

Comment:
do we want default to be `true`?

Reply:
It should be true, otherwise you risk breaking dynamic import for apps that already utilize it.